api/v1/openapi.yaml

```diff
@@ -942,6 +946,24 @@ definitions:
         type: array
         items:
           type: string
+  MonitorStatus:
+    description: Status of the node monitor
+    properties:
+      cpus:
+        description: Number of CPUs to listen on for events.
+        type: integer
+      npages:
+        description: Number of pages used for the perf ring buffer.
+        type: integer
+      pagesize:
+        description: Pages size used for the perf ring buffer.
+        type: integer
+      lost:
+        description: Recorded sample number lost by perf.
```

Comment:
Is this the number of samples lost/dropped by perf? If yes, please update this to "Number of samples lost by perf."